teachers_digital_platform/css/organisms/well.less

```diff
@@ -0,0 +1,36 @@
+// Well styling on Activity search page
+
+.o-well {
+    .tdp-activity-search & {
+        background-color: transparent;
+        max-width: none;
+
+        .h5 {
+            font-size: .875rem;
+        }
+
+        .h6 {
+            font-size: 1rem;
+            text-transform: none;
+        }
+
+        .layout-row > * {
+            display: inline;
+        }
+
+        .activity__intro {
+            font-size: 1.125rem;
+        }
+    
+        .activity__meta-wrapper {
```

Comment:
Double-underscores are for modifiers, so this should be `.activity_meta-wrapper`.

Reply:
Fixed.